framework/wazuh/cluster/local_client.py

```diff
@@ -119,6 +119,8 @@ def __init__(self, command: bytes, data: bytes, wait_for_complete: bool):
                                              path='{}/queue/cluster/c-internal.sock'.format(common.ossec_path))
         except ConnectionRefusedError:
             raise exception.WazuhException(3012)
+        except MemoryError as e:
+            raise exception.WazuhException(3012, str(e))

```

Comment:
I think the text of this exception (3012) does not tell the user what is really happening. I should create another exception to tell explicitly that '/tmp' folder needs permissions.

Reply:
Ok, I'm going to create a new exception.